tests/test_layout_padded_left.cpp

```diff
@@ -0,0 +1,478 @@
+#define MDSPAN_INTERNAL_TEST
+#define _MDSPAN_DEBUG
+#include <cassert>
+
+#include <mdspan/mdspan.hpp>
+#include <gtest/gtest.h>
+
+namespace KokkosEx = MDSPAN_IMPL_STANDARD_NAMESPACE::MDSPAN_IMPL_PROPOSED_NAMESPACE;
+
+// Compile time tests
+
+// For internal traits
+struct fake_mapping {
+  using layout_type = Kokkos::Experimental::layout_left_padded<5>;
+};
+
+static_assert(!Kokkos::Experimental::detail::__is_layout_left_padded_mapping<
+              fake_mapping>::value);
+
+static_assert(Kokkos::Experimental::detail::__is_layout_left_padded_mapping<
+              Kokkos::Experimental::layout_left_padded<4>::mapping<
+                  Kokkos::extents<size_t, 4, 7>>>::value);
+
+// layout_left_padded must be trivial
+static_assert(std::is_trivial_v<KokkosEx::layout_left_padded<0>>);
+static_assert(std::is_trivial_v<KokkosEx::layout_left_padded<4>>);
+static_assert(std::is_trivial_v<KokkosEx::layout_left_padded<Kokkos::dynamic_extent>>);
+
+// actual padding stride
+// If extents_type::rank() equals zero or one, then 0.
+static_assert(KokkosEx::layout_left_padded<0>::mapping<Kokkos::extents<std::size_t, 0>>::__actual_padding_value == 0);
```

Comment:
```suggestion
static_assert(KokkosEx::layout_left_padded<0>::mapping<Kokkos::extents<std::size_t, 0>>::__actual_padding_value == 0);
static_assert(KokkosEx::layout_left_padded<0>::mapping<Kokkos::extents<std::size_t>>::__actual_padding_value == 0);
```
zero padding with rank-0